src/Raven.Studio/typescript/viewmodels/shell/about.ts

```diff
@@ -171,15 +172,23 @@ class about extends viewModelBase {
     canRenewLicense = ko.pureComputed(() => {
         return this.licenseType() === 'Developer' || this.licenseType() === 'Community';
     });
-    
-    licenseAttribute(name: keyof Raven.Server.Commercial.LicenseStatus) {
-        return ko.pureComputed(() => {
-           const licenseStatus = license.licenseStatus();
-           if (licenseStatus) {
-               return licenseStatus[name] ? "icon-checkmark" : "icon-cancel";
-           }
-           return "icon-cancel";
-        });
+
+    formatLicenseStatusValue(
+        name: Exclude<keyof Raven.Server.Commercial.LicenseStatus, "Attributes" | "Version">
+    ): string | number {
+        const value = license.getStatusValue(name);
```

Comment:
what if license doesn't have given attribute. Will that display as infinity?

Reply:
Yes, but boolean fields are required, so there will be no problem. We expect unfilled fields only in numeric values ​​(and then it means infinity)

Reply:
Yes, but boolean fields are required, so there will be no problem. We expect unfilled fields only in numeric values ​​(and then it means infinity)